### PR TITLE
[FIX] default_warehouse_from_sale_team: pre-commit-vauxoo error

### PR DESCRIPTION
**Pull request by montesinose**
Fix lint: - (implicit-str-concat)

### CODE REVIEW

**Comment by montesinose** [comment]
Please review @luisg123v

File: search_action_domain/README.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Search Action Domain
 ====================
 
 This module lets the user to use server actions as a filters in the "Favorites"
-section. In other words, the user might use complex queries to search the records.  
+section. In other words, the user might use complex queries to search the records.
```
Comment (luisg123v) [inline search_action_domain/README.rst:8]: Apply this in a separated commit, as it's a different module. This same PR so we can see it's green

**Review by luisg123v — CHANGES_REQUESTED** [review]
Commit description: I think there's a missing new line after colon, is it intentional?

**Comment by montesinose** [comment]
> Commit description: I think there's a missing new line after colon, is it intentional?

It is. Do you want me to change it?

**Comment by luisg123v** [comment]
No, if it looks good to you, then I won't object.

**Review by luisg123v — APPROVED** [review]

**Pull request merged**